packages/babel-preset-cozy-app/index.js

```diff
@@ -94,7 +102,7 @@ const mkConfig = (api, options) => {
       }
     ]
   ]
-  if (!node && transformRegenerator) {
```

Comment:
Should we provide a way to sill be able to disable transform runtime here?
[Cozy-client](https://github.com/cozy/cozy-client/blob/master/.babelrc#L2) (and maybe some other libraries) seem to disable it but maybe we should not have to disable it for browser environment